src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -308,22 +312,45 @@ class SeekBar extends Slider {
   }
 
   /**
-   * Called when this SeekBar has focus and a key gets pressed down. By
-   * default it will call `this.handleAction` when the key is space or enter.
+   * Called when this SeekBar has focus and a key gets pressed down.
+   * Supports the following keys:
+   *
+   *   Space or Enter key fire a click event
```

Comment:
should these extra functions be on the slider itself? similar to how arrow keys call `stepBack` or `stepForward`? Would this functionality be helpful when focused on the volume slider as well?

Reply:
Support for the additional keys is not expected on the volume slider; there's already a mute button to do much of what the additional keys here do, and it's only 10 presses of Up Arrow or Down Arrow to move full-range on the volume control. I *guess* that support for 0..9 might be nice, but it's not available on any general purpose sliders, nor or YouTube's volume control.

Reply:
Sounds good